<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#131239;</string>
  </property>
  <widget class="QPushButton" name="f_back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Georgia&quot;;
background:white;
border-radius:5px;
color:black;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images and icon/Back.png</normaloff>../Images and icon/Back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeading">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>500</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Light</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Copperplate Gothic Light&quot;;
color:white;

</string>
   </property>
   <property name="text">
    <string>LEADERBOARD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>740</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgreen;</string>
   </property>
   <column>
    <property name="text">
     <string>Username</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Score</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
